torch_xla/csrc/token_handler.h

```diff
@@ -1,4 +1,5 @@
-#pragma once
+#ifndef XLA_TORCH_XLA_CSRC_TOKEN_HANDLER_H_
+#define XLA_TORCH_XLA_CSRC_TOKEN_HANDLER_H_
```

Comment:
I wonder why we need this `#define XLA_TORCH_XLA_CSRC_XLA_GRAPH_EXECUTOR_H_` macro.